r/tools/linuxlibs.R

```diff
@@ -0,0 +1,341 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+args <- commandArgs(TRUE)
+VERSION <- args[1]
+dst_dir <- paste0("libarrow/arrow-", VERSION)
```

Comment:
Does it requires the distribution token in the path? I foresee a problem with invoking multiple docker images on the same master, e.g. on a developer machine.

Reply:
It's arbitrary, but if I change it here, I have to change it in the `configure` script too, and that would mean I'd need to determine the distro/version in the configure script too, and personally I'd prefer to avoid rewriting that logic in bash/sed.

Can we wait to see if what you're fearing is a problem? If it does happen, the package installation should fail pretty fast, and deleting the previously built dir is a quick fix.